docs/metadata.md

```diff
@@ -1200,7 +1200,7 @@ ___
     !!! warning "Only available with dbt config-version: 1"
 
     ```yaml
-    PIT_CUSTOMER:
+    pit_customer:
       vars:
```

Comment:
We no longer use vars in a `dbt_project.yml` Please apply the style guide/template for tutorials here.

Reply:
Resolved